src/scan.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-SHARED_DIR=${CISA_HOME}/shared
+SHARED_DIR="${CISA_HOME}"/shared
 
 echo "Creating artifacts folder..."
```

Comment:
In https://github.com/cisagov/saver/commit/7d784b037eaeb391a3ed3b47c4b8100a2aa6bf8f you remove quotes like these as unnecessary. I would like us to be consistent so would you please update this project or that one?

Reply:
That commit is a different case.  Those are `echo` statements that contain no shell variables.  Maybe you meant to refer to a different commit?

The shell variables have to be wrapped in quotes to satisfy our shell linter, no?

Reply:
This comment is attached to line 5 which is an `echo` statement with no variables. The preview is just including the other bits above it for context.